http2/src/test/java/com/blazemeter/jmeter/http2/sampler/HTTP2StreamHandlerTest.java

```diff
@@ -239,4 +240,18 @@ public void onDataTest2() {
     assertTrue(resSR.isSuccessful());
   }
 
-} 
+  @Test
+  public void onResetTest() {
+
+    ResetFrame resetFrame = Mockito.mock(ResetFrame.class);

```

Comment:
It is easier in this case to just create an instance of the class instead of mock it. I.e:

```java
ResetFrame resetFrame = new ResetFrame(0, ErrorCode.REFUSED_STREAM_ERROR.code);
```